crates/nargo_cli/tests/execution_success/references/src/main.nr

```diff
@@ -35,6 +35,12 @@ fn main(mut x: Field) {
     regression_2054();
     regression_2030();
     regression_2255();
+
+    // x == 3 at this point
+    regression_2218_if_inner_if(x, 10);
+    regression_2218_if_inner_else(20, x);
+    regression_2218_else(x, 3);
+    regression_2218_loop(x, 10);
```

Comment:
Shouldn't these arguments be inside `if_inner_if`, etc if we want to guarantee it takes the inner_if path?

Reply:
Sorry not sure what you mean here do you mean the loop test inside of the others?

Reply:
I mean `regression_2218_if_inner_if();` Then remove the `x` and `y` parameters inside that function and replace them with `x` and `10` to guarantee that function takes the correct path in `regreesion_2218(..)`

Reply:
Since as it is right now you can do `regression_2218_if_inner_if(20, 20)` and not get the if_inner_if path.

Reply:
Ah, fair enough. I missed the witness part.

Reply:
I included an assert to guarantee we hit the if statements we expect